library/src/window_impl.cxx

```diff
@@ -153,6 +153,10 @@ class window_impl::internals
 window_impl::window_impl(const options& options, Type type)
   : Internals(std::make_unique<window_impl::internals>(options))
 {
+#if defined(_WIN32) && (WINVER >= 0x0605)
+  SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
```

Comment:
maybe only for native ? I dont think its our responsability to set that otherwise ?